src/types/index.ts

```diff
@@ -182,7 +182,8 @@ declare global {
           "block-uid": string;
         };
         components: {
-          renderBlock: (args: { uid: string; el: HTMLElement }) => null;
+          renderBlock: (args: { uid: string; el: HTMLElement; zoomPath?: boolean }) => null;
```

Comment:
should these be `=> Promise<void>` ?
https://roamresearch.com/#/app/roamjs-manager/page/36gmOb5oT

Reply:
yup!

In general, the types here should always match "real life" as much as possible